common/com/twitter/intellij/pants/PantsBundle.properties

```
@@ -56,3 +56,14 @@ pants.resolve.incremental.import.unsupported=No target root found for constructi
 pants.settings.text.use.intellij.compiler=Use the IntelliJ compiler [Experimental] This aims to reduce the iteration overhead by interacting with IntelliJ's compiler directly.
 pants.settings.text.use.intellij.compiler.help.messasge=If your project does not work with IntelliJ compiler out of the box, please try to conform the code structure to 1:1:1.
 pants.settings.text.use.intellij.compiler.help.messasge.link=https://www.pantsbuild.org/build_files.html#target-granularity
+
+pants.bsp.select.targets=Amend BSP project
+pants.bsp.error.failed.to.fetch.targets=Failed to fetch imported targets list, please look at the logs for more details
+pants.bsp.error.failed.to.fetch.targets.in.dir=Failed to fetch list of targets
+pants.bsp.error.action.not.supported.title=Action not supported
+pants.bsp.error.failed.more.than.one.bsp.project.not.supported.message=Amending targets not supported for multi-BSP root projects
+pants.bsp.error.failed.not.a.bsp.pants.project.message=This project is not a Pants-BSP one
```

I'm not sure where this message would appear, is the command visible in regular Pants projects?
```suggestion
pants.bsp.error.failed.not.a.bsp.pants.project.message=Amending targets is not supported for non-BSP projects
```

---

Right, I just made it hidden for non-BSP projects. I think the message box can stay here, just in case.